src/main/java/com/thealgorithms/misc/MirrorOfMatrix.java

```diff
@@ -0,0 +1,45 @@
+package com.thealgorithms.misc;
+
+import java.util.Scanner;
+
+// Problem Statement
+/*
+We have given an array of m x n (where m is the number of rows and n is the number of columns).
+Print the new matrix in such a way that the new matrix is the mirror image of the original matrix.
+
+The Original matrix is:   |   The Mirror matrix is:
+1	2	3	              |   3	 2	 1
+4	5	6	              |   6	 5	 4
+7	8	9	              |   9	 8	 7
+
+@author - Aman (https://github.com/Aman28801)
+*/
```

Comment:
Please update the description of the problem. Try to explain in what sentence what your method does.

Reply:
done

Reply:
Please write a proper documentation of your class. The method `Mirror` does not print anything.